src/components/auth/authorization-popup.js

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default class AuthorizationPopup extends React.Component {
+  close =() => {
```

Comment:
```suggestion
  close = () => {
```
Maybe run a linter, just some small nits